nomad/grapl-core.nomad

```diff
@@ -1131,12 +1163,11 @@ job "grapl-core" {
         PLUGIN_EXECUTION_IMAGE                          = var.container_images["generator-execution-sidecar"] # TODO: add support for analyzer too
         PLUGIN_REGISTRY_BUCKET_AWS_ACCOUNT_ID           = var.plugin_registry_bucket_aws_account_id
         PLUGIN_REGISTRY_BUCKET_NAME                     = var.plugin_registry_bucket_name
+        PLUGIN_EXECUTION_OBSERVABILITY_ENV_VARS         = var.observability_env_vars
```

Comment:
Why does this need a separate env var that just repeats the existing env var?

Reply:
Ah, this is passed into the plugins. Got it.

Reply:
So, we pass through some vars from
`grapl-core.nomad` -> plugin-registry -> `hax_docker_plugin.nomad` -> plugin binaries

what we're doing to plugin-registry is:
- passing it env vars: `OTEL_BLAH_BLAH=1, ...` that plugin-registry uses at runtime
- passing it a template string: "OTEL_BLAH_BLAH=1\n..." that plugin-registry passes down to the plugin binaries

By passing in the template string, we basically **remove some complexity around the optional-ness of the env vars** (the reason we did the aws_env_vars_for_local stuff in the first place) + make hax_docker_plugin.nomad more-resemble grapl-core.nomad